views/error/415.php

```diff
@@ -0,0 +1,16 @@
+<!doctype html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
+          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Document</title>
+</head>
+<body>
+    <h1>415 ERROR</h1>
+
+    <?= $message ?>
```

Comment:
you should escape this, I know we haven't covered that in the series yet so not a big deal but should use htmlspecialchars. XSS will be covered in the series